packages/yoastseo/spec/scoring/assessments/readability/SubheadingDistributionTooLongAssessmentSpec.js

```diff
@@ -447,107 +461,6 @@ describe( "A test for the assessment applicability", () => {
-	// 		" <a href='https://yoa.st/34y' target='_blank'>Add subheadings to improve readability</a>." );
-	// } );
-	//
-	// it( "Scores a long text in Japanese (>600 characters), which has subheadings and some sections of the" +
-	// 	" text are above 700 characters long.", function() {
-	// 	const paper = new Paper( shortTextJapanese + subheading + veryLongTextJapanese + subheading + veryLongTextJapanese );
-	// 	const japaneseResearcher = new JapaneseResearcher( paper );
-	// 	const subheadingDistributionTooLongJA = new SubheadingDistributionTooLong();
-	// 	const results = subheadingDistributionTooLongJA.getResult( paper, japaneseResearcher );
-	// 	expect( results.getScore() ).toBe( 3 );
-	// 	expect( results.getText() ).toBe( "<a href='https://yoa.st/34x' target='_blank'>Subheading distribution</a>: " +
-	// 		"2 sections of your text are longer than 600 characters and are not separated by any subheadings." +
-	// 		" <a href='https://yoa.st/34y' target='_blank'>Add subheadings to improve readability</a>." );
-	// } );
-	//
-	// it( "Scores a long text in Japanese (>600 characters), which has subheadings and all sections of the " +
-	// 	"text are <600 characters, except for one, which is above 700 characters long.", function() {
-	// 	const paper = new Paper( shortTextJapanese + subheading + veryLongTextJapanese + subheading + shortTextJapanese );
-	// 	const japaneseResearcher = new JapaneseResearcher( paper );
-	// 	const subheadingDistributionTooLongJA = new SubheadingDistributionTooLong();
-	// 	const results = subheadingDistributionTooLongJA.getResult( paper, japaneseResearcher );
-	// 	expect( results.getScore() ).toBe( 3 );
-	// 	expect( results.getText() ).toBe( "<a href='https://yoa.st/34x' target='_blank'>Subheading distribution</a>: " +
-	// 		"1 section of your text is longer than 600 characters and is not separated by any subheadings." +
-	// 		" <a href='https://yoa.st/34y' target='_blank'>Add subheadings to improve readability</a>." );
-	// } );
-	//
-	// it( "Scores a long text (>600 characters), which has subheadings and some sections of the text are " +
-	// 	"above 700 characters long.", function() {
-	// 	const paper = new Paper( shortTextJapanese + subheading + veryLongTextJapanese + subheading + veryLongTextJapanese );
-	// 	const japaneseResearcher = new JapaneseResearcher( paper );
-	// 	const subheadingDistributionTooLongJA = new SubheadingDistributionTooLong();
-	// 	const results = subheadingDistributionTooLongJA.getResult( paper, japaneseResearcher );
-	// 	expect( results.getScore( paper, japaneseResearcher ) ).toBe( 3 );
-	// 	expect( results.getText( paper, japaneseResearcher ) ).toBe( "<a href='https://yoa.st/34x' target='_blank'>" +
-	// 		"Subheading distribution</a>: 2 sections of your text are longer than 600 characters and are not separated " +
-	// 		"by any subheadings. <a href='https://yoa.st/34y' target='_blank'>Add subheadings to improve readability</a>." );
-	// } );
```

Comment:
I removed this since it's already covered under this test:

> describe( "A test for scoring too long text fragments without a subheading for languages that use 'characters' in the feedback strings instead of 'words'", function() {

In terms of tests that cover Japanese specific configuration, they are already covered in this test case:

> describe( "Language-specific configuration for specific types of content is used: Japanese", function() {